modules/social_features/social_post/templates/post--featured.html.twig

```diff
@@ -53,8 +53,14 @@
           {% endif %}
 
           <div class="clearfix"></div>
+
+          <div class="album-name">
+            {{ content.field_album }}
+          </div>
+
           <div class="media-post-social">
             {{ content.like_and_dislike }}
+
             {% if dashboard and comment_count > 0 %}
```

Comment:
There is no way to override templates in `social_dashboard` :( So, this is the reason why we have conditions to check the `dashboard` variable.